test/test_admin_script.py

```diff
@@ -6,9 +6,11 @@
 from gdcdatamodel import gdc_postgres_admin as pgadmin
 from gdcdatamodel import models
 
+from test.helpers import DB_CONFIG
```

Comment:
Optional suggestion for this and the imports in the other test files.
```suggestion
from test.helpers import DB_CONFIG
```